scripts/cnv_cromwell_tests/germline/run_cnv_germline_workflows.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -l
+set -e
+#cd in the directory of the script in order to use relative paths
+script_path=$( cd "$(dirname "${BASH_SOURCE}")" ; pwd -P )
+cd "$script_path"
+
+ln -fs /home/travis/build/broadinstitute/gatk-protected/scripts/cnv_wdl/cnv_common_tasks.wdl
+ln -fs /home/travis/build/broadinstitute/gatk-protected/scripts/cnv_wdl/germline/gCNV_panel_creation_workflow.wdl
+ln -fs /home/travis/build/broadinstitute/gatk-protected/scripts/cnv_wdl/germline/gCNV_single_sample_calling_workflow.wdl
+ln -fs /home/travis/build/broadinstitute/gatk-protected/scripts/cnv_wdl/germline/gCNV_cohort_calling_workflow.wdl
+
+CROMWELL_JAR="cromwell-0.26.jar"
+
+# Panel WES
+java -jar ~/${CROMWELL_JAR} run /home/travis/build/broadinstitute/gatk-protected/scripts/cnv_wdl/germline/gCNV_panel_creation_workflow.wdl gCNV_panel_creation_workflow_wes.json
```

Comment:
Let's change the filenames of WDL workflows/templates to start with `cnv_germline_` rather than `gCNV_`, to be more analogous to the somatic filenames.  But you can keep "gCNV" in tasknames, etc. for now.